### ISSUE TITLE
Some decorative statues do not have a recipe

### ISSUE THREAD

**Issue by darkphan** [Darkosto/SkyFactory4Issues#196]
The following decorative statues do not have a recipe:

- decorative flood statue
- decorative king cluck statue
- decorative chicken jockey
- decorative pufferfish
- decorative wasteland pig 
- decorative campfire statue

Also, we do not seem to have a method of getting the totem of undying to craft the decorative evoker statue.

**Comment by Mrbysco** [comment]
As the developer of Statues they aren't supposed to have recipes. There's in-world crafting to get them. It's just not well-known.

**Comment by darkphan** [comment]
Ah ok. So really the only issue then is the totem of undying?

**Comment by svennieke** [comment]
The totem of undying drops from the mob that can drop the real totem too ;)

well that is if you mean the totem of undying statue and not the real totem of undying.

**Comment by darkphan** [comment]
Yeah but the evokers wont spawn in the world, since there are no mansions. However I think that evoker seeds are being added so it should be good now.

**Comment by svennieke** [comment]
Actually that is a good point, i will look in to a mechanic to implement in to statues that allows the possible creation of a evoker or the totem of undying statue
